sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql

```diff
@@ -91,13 +91,13 @@ create table t2 as (select * from t1);
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-create table if not exists t2 (select * from (select * from t1 union select * from t1 ) tt);
+create table if not exists t2; (select * from (select * from t1 union select * from t1 ) tt);
```

Comment:
create table if not exists t2 AS (select * from (select * from t1 union select * from t1 ) tt);

Reply:
The review has been reflected.